src/registry_schemas/schemas/party.json

```diff
@@ -56,11 +56,19 @@
                 "person": {
                     "$ref": "#/definitions/person"
                 },
-                "deliveryAddress": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
-                },
-                "mailingAddress": {
-                    "$ref": "https://bcrs.gov.bc.ca/.well_known/schemas/address"
+                "address": {
```

Comment:
I've seen this structure where the addresses are wrapped in an "address" object, but I don't know if this necessarily makes sense. If we use the initial director schema as an example, it had a mailingAddress and deliveryAddress as standalone properties. I don't disagree with this alternate structure but I do think we should be consistent.

Reply:
For sure, that makes sense, i agree in consistency. I know that Directors are going away as a whole ( eventually ), being replaced with persons? In the new design is there an address wrapper? 

In this new structure, the addresses don't exist on the 'Person' object, so i'm assuming there will be a lookup between Person and Address?

Reply:
I guess I should have also mentioned the person also does not have an address wrapper (according to what I can see in entity filer, and legal api)

Reply:
I thought the idea was to have the addresses follow the same structure throughout the incorporations flow ( for offices, person etc). Also if we plan to make use of the same filer, I agree that it has to be consistent. But if the filer is going to be different then we can use a different structure. Thoughts?

Reply:
so for example in the entity filer, instances of:
director_info['deliveryAddress']

would have to change to:
director_info['address']['deliveryAddress']

correct me if I'm wrong @lekshmimallika-aot